Postman/local.postman_environment.json

```diff
@@ -34,12 +34,12 @@
 		},
 		{
 			"key": "projectId",
-			"value": "24",
+			"value": 37,
 			"enabled": true
 		},
 		{
 			"key": "userName",
-			"value": "Developer",
+			"value": "postmantest_username_updated",
```

Comment:
I don't think these were meant to be changed. This goes for all changes in local.postman_environment.json
Please consider discarding changes in here.